test/CMakeLists.txt

```diff
@@ -211,18 +211,22 @@ if(MIOPEN_TEST_WITH_MIOPENDRIVER)
     add_dependencies(check MIOpenDriver)
 endif()
 
+
+set(MIOPEN_TEST_SUFFIX)
```

Comment:
```suggestion
set(MIOPEN_GTEST_SUFFIX)
```

Reply:
At some point there will be only gtests for the tests. It means that currently having the other tests is just a coincidence and everything related to gtests should be related to simple tests and I don't think that explicit `gtest` makes much sense here.
But actually I don't care.